pyinfra/modules/pip.py

```diff
@@ -51,7 +51,7 @@ def virtualenv(
 def packages(
     state, host,
     packages=None, present=True, latest=False,
-    requirements=None, virtualenv=None
+    requirements=None, virtualenv_root=None
```

Comment:
I'm not keen on renaming this arg as it'll break backwards compatibility - I've worked around the conflict in the `apt` and `yum` modules by creating a second reference, eg. `_virtualenv = virtualenv` (https://github.com/Fizzadar/pyinfra/blob/develop/pyinfra/modules/yum.py#L146-L173).